.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  issue_comment:
+    name: Auto build image
+    if: startswith(github.event.comment.body, '/docs')
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: npm install
+        working-directory: docs/site
+      - name: Static website hosting
+        run: npm run build
+        working-directory: docs/site
+      - name: Build static to oss
```

Comment:
Similar to the `OSS -staticPages` Workflow, it can be merged

Reply:
ok